client/allocdir/task_dir.go

```diff
@@ -142,15 +141,9 @@ func (t *TaskDir) Build(chrootCreated bool, chroot map[string]string, fsi cstruc
 // buildChroot takes a mapping of absolute directory or file paths on the host
 // to their intended, relative location within the task directory. This
 // attempts hardlink and then defaults to copying. If the path exists on the
-// host and can't be embedded an error is returned. If chrootCreated is true
-// skip expensive embedding operations and only ephemeral operations (eg
-// mounting /dev) are done.
-func (t *TaskDir) buildChroot(chrootCreated bool, entries map[string]string) error {
-	if !chrootCreated {
-		// Link/copy chroot entries
-		return t.embedDirs(entries)
-	}
-	return nil
+// host and can't be embedded an error is returned.
+func (t *TaskDir) buildChroot(entries map[string]string) error {
```

Comment:
This func seems like a useless wrapper now.

Reply:
Indeed, I liked having it around, as the function name and comment is useful.  `embedDirs()` doesn't convey the same thing.